rocAL/rocAL/source/meta_data/meta_node_resize_mirror_normalize.cpp

```diff
@@ -90,6 +90,11 @@ void ResizeMirrorNormalizeMetaNode::update_parameters(pMetaDataBatch input_meta_
             bb_coords.push_back(coords_buf[j]);
             bb_labels.push_back(labels_buf[j]);
         }
+        // get roi width and height of output image
+        auto img_roi_size = input_meta_data->get_img_roi_sizes_batch()[i];
```

Comment:
Will this be updated in other nodes? or not required?

Reply:
Not required unless any other training requires ROI info for the final augmentation